scripts/hello_world.py

```diff
@@ -21,8 +21,9 @@ def Main():
 	Target means run this function when a thread is initiated."""
 	myThread = threading.Thread(target=MyFunction) 	
 	myThread.start() 	# Starting a thread
-	fp = open("def.txt")
-	fp.close()
+	try:
+		fp = open("def.txt")
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
This line of code might contain a resource leak. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the following resource: *fp*. The resource is allocated by call *builtins.open*. Execution paths that do not contain closure statements were detected. To prevent this resource leak, close *fp* in a try-finally block or declare it using a `with` statement.

**More info**
[View details about the `with` statement in the Python developer&#39;s guide](https://www.python.org/dev/peps/pep-0343/) (external link).